### DISCUSSION TITLE
How to position the camera in Carla

### DISCUSSION THREAD

**Discussion by thhung** [carla-simulator/carla#6770] [Q&A]
Assume that I have a map and some of vehicles that I want to put in the automode. I want to record this road from a fixed camera on the side of the road. To do that, I would need the position and transformation to set the position of camera. So I have some ideas about the position of the camera relative to the road but not the car. However, I cannot find the position of the road in the map so I don't know how to set the camera position. Could someone help me some simpler way to solve this issue?

**Comment by XiaoFei9704** [comment]
There's a free online opendrive viewing tool: https://odrviewer.io/
![1697025814757_E1590C97-D5CC-4939-B7C1-EFEBF2EB0F21](https://github.com/carla-simulator/carla/assets/28669749/3ba1e0a6-e267-4567-961b-3dbcad899a25)
You can directly know the coordinates of the road.
